src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2727,7 +2727,7 @@ private string GenerateCertificateThumbprintParameter()
 
     $clientSideParameters = @{}
 
-    $parametersToLeaveRemote = 'ErrorAction', 'WarningAction', 'InformationAction'
+    $parametersToLeaveRemote = 'ErrorAction', 'WarningAction', 'VerboseAction', 'DebugAction', 'InformationAction', 'ProgressAction'
```

Comment:
I'd sort the list for better readability. 
Below there is a lot of places where we could sort the names too.

Reply:
Agreed. Applied sorting in many places that common parameters are used, and committed just now.